pkg/asset/installconfig/installconfig.go

```diff
@@ -145,3 +152,20 @@ func (a *InstallConfig) finish(filename string) error {
 	}
 	return nil
 }
+
+// convert converts possibly older versions of the install config to
+// the current version, relocating deprecated fields.
+func (a *InstallConfig) convert() error {
```

Comment:
this functions was dropped in https://github.com/openshift/installer/pull/2512 and looks like it was added as part of rebase again. we can drop this.